pom.xml

```diff
@@ -114,7 +114,7 @@
     <cloudresourcemanager.version>v1-rev6-1.22.0</cloudresourcemanager.version>
     <pubsubgrpc.version>0.1.18</pubsubgrpc.version>
     <clouddebugger.version>v2-rev8-1.22.0</clouddebugger.version>
-    <dataflow.version>v1b3-rev221-1.22.0</dataflow.version>
+    <dataflow.version>v1b3-rev235-1.22.0</dataflow.version>
```

Comment:
Please update the build.gradle file as well.

Reply:
Done. Thanks for the catch.